spec/index.html

```diff
@@ -4240,6 +4240,11 @@ <h3>CONSTRUCT</h3>
           included in the output RDF graph. The graph template can contain triples with no variables
           (known as ground or explicit triples), and these also appear in the output RDF graph returned
           by the CONSTRUCT query form.</p>
+        <p class="note">
+          Note that the construction of the result graph by "set union" does not
```

Comment:
Since the note is clear already:
```suggestion
          The construction of the result graph by "set union" does not
```